modules/test/playwright/tests/portal-search-web/customFilter.spec.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Page, expect, mergeTests} from '@playwright/test';
+
+import {dataApiHelpersTest} from '../../fixtures/dataApiHelpersTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {isolatedSiteTest} from '../../fixtures/isolatedSiteTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {pageEditorPagesTest} from '../../fixtures/pageEditorPagesTest';
+import {pagesAdminPagesTest} from '../../fixtures/pagesAdminPagesTest';
+import {searchPageTest} from '../../fixtures/searchPageTest';
+import getRandomString from '../../utils/getRandomString';
+import getBasicWebContentStructureId from '../../utils/structured-content/getBasicWebContentStructureId';
+import {pagesPagesTest} from '../layout-admin-web/fixtures/pagesPagesTest';
+
+export const test = mergeTests(
+	dataApiHelpersTest,
+	isolatedSiteTest,
+	loginTest(),
+	featureFlagsTest({
+		'LPS-178052': true,
+	}),
+	pagesAdminPagesTest,
+	pagesPagesTest,
+	searchPageTest,
+	pageEditorPagesTest
+);
+
+test.describe('Custom Filter', () => {
+	test('Custom Filter Configuration Dont Leak Between Experiences @LPD-35585', async ({
```

Comment:
It looks like for these test messages, other tests will lowercase the words as if it were a sentence, so let’s go with that then.

So it would be (small grammar nitpick): `Custom filter configuration doesn't leak between experiences @LPD-35585`